main.tf

```diff
@@ -196,8 +196,10 @@ resource "aws_iam_role_policy" "ses" {
 resource "aws_sns_topic" "ses_bounces" {
   count = var.create_bounce_topic ? 1 : 0
 
-  kms_master_key_id = "alias/aws/sns"
```

Comment:
It seems counter intuitive that to decrypt the topic you remove the key. Or is it that you're disabling encryption by removing it?